environment-api/src/main/java/com/sequenceiq/environment/api/v1/credential/endpoint/CredentialEndpoint.java

```diff
@@ -78,7 +78,7 @@ public interface CredentialEndpoint {
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = CredentialOpDescription.CREATE, produces = MediaType.APPLICATION_JSON, notes = CredentialDescriptor.CREDENTIAL_NOTES,
             nickname = "createCredentialV1", httpMethod = "POST")
-    CredentialResponse post(@Valid CredentialRequest request);
+    CredentialResponse create(@Valid CredentialRequest request);
```

Comment:
I think this requires cdpcli change.

Reply:
Yes, we need to refresh that one as well, but since the rest API is compatible therefore it is OK.